package.json

```diff
@@ -43,9 +43,10 @@
   },
   "homepage": "https://github.com/mozilla/addons-linter#readme",
   "dependencies": {
+    "@babel/eslint-parser": "7.15.8",
```

Comment:
It looks that "babel-eslint" used to be a dev dependency, while "@babel/eslint-parser" is listed as a direct dependency, 

Based on the content of this PR it looks that @babel/eslint-parser it is still only used while running eslint on the addons-linter sources for development purpose, and it may still not be needed as a strict dependency for an addons-linter user.

is there something else that I may be missing to notice?

Reply:
No, that was a mistake. Good catch, thanks.